codecov_auth/tests/unit/views/test_okta.py

```diff
@@ -10,8 +10,9 @@
 
 from codecov_auth.models import OktaUser
 from codecov_auth.tests.factories import OktaUserFactory, OwnerFactory, UserFactory
-from codecov_auth.views.okta import OktaLoginView, validate_id_token
+from codecov_auth.views.okta import OktaLoginView
 from codecov_auth.views.okta import auth as okta_basic_auth
+from codecov_auth.views.okta import validate_id_token
```

Comment:
how come we split this to a separate line?

Reply:
Uh ruff did that..